### PR TITLE
feat/improve oss page UI and screenshots 343

### LINKED ISSUE #343: Improve OSS projects page UI & Screenshots
Inconsistent format for different projects and outdated screenshots

https://guide.eoscostarica.io/docs/open-source-projects

what about using cards instead

- [ ] hacer lista de proyectos 
- [ ] actualizar layout (diseño)
- [ ] actulizar contenido

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #343 

### Steps to test
1. Run `yarn start`
1. Go to `localhost:3000`
1. Go to Developers Guidelines > Open Source Projects 

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/blockchain-security.md [hunk 1]
```diff
@@ -5,4 +5,10 @@ sidebar_label: Security
 ---
 ## Exploiting vulnerabilities in Blockchain
 
+The blockchain is a set of emerging technologies that are presented as alternatives to centralized systems. A notable example is that they seek to replace traditional banking systems through the use of cryptocurrencies without centralized control, as in the case of the well-known Bitcoin. They also have other important applications, such as in the traceability of assets, the control of corruption, and even democratic systems. Some of these capabilities are implemented through "smart contracts", basically software that runs on blockchains.
```
Comment (xavier506) [inline docs/eos-learn/blockchain-security.md:8]: Blockchain not "The blockchain" 

did we already correct this?

Comment (hanakoori01) [reply docs/eos-learn/blockchain-security.md:8]: I think we had, it may be a problem with my local branch

Comment (xavier506) [reply docs/eos-learn/blockchain-security.md:9]: seems like you have a merge conflict

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**